stanford_events_importer.install

```diff
@@ -760,3 +762,72 @@ function stanford_events_importer_update_7304(&$sandbox) {
     field_cache_clear();
   }
 }
+
+/**
+ * Update variables to point to events-legacy.stanford.edu.
+ */
+function stanford_events_importer_update_7305(&$sandbox) {
+  $category_source = variable_get('stanford_events_category_source');
+  if ($category_source == "http://events.stanford.edu/xml/drupal/v2.php?category-list") {
+    variable_set('stanford_events_category_source', 'https://events-legacy.stanford.edu/xml/drupal/v2.php?category-list');
+    watchdog('stanford_events_importer', 'Stanford Events Importer category list updated to pull from events-legacy.stanford.edu');
+  }
+  $organization_source = variable_get('stanford_events_organization_source');
+  if ($organization_source == 'http://events.stanford.edu/xml/drupal/v2.php?organization-list') {
+    variable_set('stanford_events_organization_source', 'https://events-legacy.stanford.edu/xml/drupal/v2.php?organization-list');
+    watchdog('stanford_events_importer', 'Stanford Events Importer organization list updated to pull from events-legacy.stanford.edu');
+  }
+}
+
+/**
+ * Update feeds_source config to point to events-legacy.stanford.edu.
+ */
+function stanford_events_importer_update_7306(&$sandbox) {
+  // See also stanford_events_importer_update_7201().
+  $configs = db_select('feeds_source', 'fs')
+    ->fields('fs',
+      array(
+        'feed_nid',
+        'config',
+        'source',
+      ))
+    ->condition('id', 'stanford_event_importer')
```

Comment:
What about a condition like `>condition('source', 'events.stanford.edu', 'LIKE')`

Reply:
shouldn't that be `condition('source', '%events.stanford.edu%', 'LIKE')`?

Reply:
Fair point. I'm less concerned about this one than the one that updates the `add_to_cal` fields. We'll see if I can get to this in the time allotted.

Reply:
Meh. The use case in which there would be a Feeds Importer where the `id` is `stanford_events_importer`, yet in which the `source` would be something other than `%events.stanford.edu%` is small enough that my boss would scold me for wasting her money on tracking it down.

Thank you for the feedback, and you're technically correct (which is the best kind of correct), yet I'm going to opt for letting the machines do the work here on this one-time operation.